fury/shaders/sdf_dec.frag

```diff
@@ -14,6 +14,40 @@ uniform mat4 WCVCMatrix;
 uniform mat4 WCMCMatrix;
 uniform mat4 VCMCMatrix;
 
+#define PI 3.14159265
+
+struct Superellipsoid
```

Comment:
This struct seems to be unnecessary as the sdFunction works without it. Can you explain why are you adding this?

Reply:
It was only for convenience, removed now